client/web/src/auth/PostSignUpPage.tsx

```diff
@@ -55,14 +64,26 @@ export const PostSignUpPage: FunctionComponent<PostSignUpPage> = ({
 
     const goToSearch = (): void => history.push(getReturnTo(location))
 
+    useEffect(() => {
+        eventLogger.log(getPostSignUpEvent('Page_Viewed'))
```

Comment:
We have [`logViewEvent`](https://sourcegraph.com/search?q=r%3A%5Egithub.com%2Fsourcegraph%2Fsourcegraph%24+logviewevent&patternType=literal) logging for pageview events
```suggestion
        eventLogger.logViewEvent(PostSignUpPage)
```